rsvg2/test/test-handle.rb

```diff
@@ -69,17 +70,10 @@ def setup_large_svg
         end
       end
 
-      def test_no_option
-        only_rsvg_version(2, 40, 3)
-        assert_raise(RSVG::Error::Failed) do
-          RSVG::Handle.new_from_file(@large_svg_path)
-        end
-      end
-
       def test_unlimited
         only_rsvg_version(2, 40, 3)
-        handle = RSVG::Handle.new_from_file(@large_svg_path,
-                                            :flags => :flag_unlimited)
+        handle = RSVG::Handle.new(:file => @large_svg_path,
+                                  :flags => :flag_unlimited)
         assert_equal([0, 0, 0.0, 0.0],
                      handle.dimensions.to_a)
```

Comment:
Umm. We need to rename `RSVG` module name to `Rsvg` module name to use the same namespace in `.gir`.